.eslintrc.js

```diff
@@ -0,0 +1,190 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "ignorePatterns": ["/*", "!/src"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "tsconfig.json",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "eslint-plugin-jsdoc",
+        "eslint-plugin-prefer-arrow",
+        "@typescript-eslint",
+        "jsdoc"
+    ],
+    "root": true,
+    "rules": {
```

Comment:
These rules are all chosen by running `tslint-to-eslint-config` package.  They are equivalent (as can be) to the tslint configuration.